client/app/queue/components/IssueRemandReasonsOptions.jsx

```diff
@@ -308,6 +312,16 @@ class IssueRemandReasonsOptions extends React.PureComponent {
       );
     }
 
+    if (this.props.featureToggles.additional_remand_reasons) {
+      this.filterSelectableAmaRemandReasons('medicalExam', 0);
+      this.filterSelectableAmaRemandReasons('medicalExam', 1);
+    } else if (!this.props.featureToggles.additional_remand_reasons) {
```

Comment:
@KiMauVA you can directly use `else` block instead of `else if (!this.props.featureToggles.additional_remand_reasons)`
```suggestion
    } else {
```